app/controllers/spree/admin/orders_controller_decorator.rb

```diff
@@ -37,9 +37,7 @@ def resend
   end
 
   def invoice
-    pdf = render_to_string pdf: "invoice-#{@order.number}.pdf",
-                           template: invoice_template,
-                           formats: [:html], encoding: "UTF-8"
+    pdf = InvoiceRenderer.new.render(@order)
```

Comment:
```suggestion
    pdf = InvoiceRenderer.new.render_to_string(@order)
```

Reply:
Yes, thanks @luisramos0. I renamed that method at some point and forgot this call.